src/lib/peers/tcp-hole-punch/TCPHolePunchServer.ts

```diff
@@ -0,0 +1,15 @@
+// taken from https://github.com/SamDecrock/node-tcp-hole-punching
+import net from 'net';
+
+class TCPHolePunchServer {
+  constructor() {
+    let server = net.createServer((socket) => {
+      console.log('> Connect to this public endpoint with clientB:', socket.remoteAddress + ':' + socket.remotePort);
+    });
+    server = server.listen(9999, () => {
```

Comment:
These ports should not be hardcoded.

Reply:
yeah the whole TCP hole punch idea was a dead end and I've removed this whole folder. None of the other ports are hardcoded except a default of `1314` for out `PK_PORT`